submissions/evgenii-del/dom/js/script.js

```diff
@@ -0,0 +1,72 @@
+const cats = [
+    {
+        name: 'Пуфик',
+        icon: 'fas fa-cat',
+        url: 'https://images.pexels.com/photos/5200916/pexels-photo-5200916.jpeg?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=800&q=60',
+    },
+    {
+        name: 'Люся',
+        icon: 'fas fa-paw',
+        url: 'https://images.pexels.com/photos/1366996/pexels-photo-1366996.jpeg?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=800&q=60',
+    },
+    {
+        name: 'Трейси',
+        icon: 'fas fa-fish',
+        url: 'https://images.pexels.com/photos/1457595/pexels-photo-1457595.jpeg?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=800&q=60',
+    },
+    {
+        name: 'Люк',
+        icon: 'fas fa-heart',
+        url: 'https://images.pexels.com/photos/2870353/pexels-photo-2870353.jpeg?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=800&q=60',
+    },
+    {
+        name: 'Спарк',
+        icon: 'fas fa-sun',
+        url: 'https://images.pexels.com/photos/979247/pexels-photo-979247.jpeg?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=800&q=60',
+    }
+]
+
+const main = document.querySelector('.js-main');
+const menu = document.querySelector('.js-menu');
+
+const renderCats = (position) => {
+    let cat = cats[position];
+    main.innerHTML = `
+        <img id="img" src=${cat.url} alt=${cat.name}>`;
+}
+
+const renderItem = (cat, index) => {
+    let block = document.createElement('li');
+    block.innerHTML = `
+        <a href="#" data-id="${index}">
+            <span class="icon">
+                <i class="${cat.icon}"></i>
+            </span>
+            <p class="text">${cat.name}</p>
+        </a>
+    `;
+    menu.appendChild(block);
+}
+
+const removeActive = () => {
```

Comment:
Your decision about removing class is ok. But as we have only one element with the `active` class, let's make it simpler. I just recommend finding this element by 
``` javascript
const activeElement = document.querySelector('a.active')
```
And then you can use `activeElement.classList.remove('active')`
Also, it's better to add some class for these links